mkosi

```diff
@@ -56,6 +56,14 @@ class OutputFormat(Enum):
     directory = 4
     subvolume = 5
     tar = 6
+    raw_xfs = 7
+
+RAW_FORMATS = [
+  OutputFormat.raw_gpt,
+  OutputFormat.raw_btrfs,
+  OutputFormat.raw_squashfs,
+  OutputFormat.raw_xfs
+]
```

Comment:
not that i was a python guru, but wouldn't it be nicer to use a tuple rather than a list for this? after all this is supposed to be immutable... i.e. [] → ()